OrbitGl/App.cpp

```diff
@@ -871,6 +871,17 @@ void OrbitApp::StopCapture() {
     return;
   }
 
+  if (metrics_uploader_ != nullptr) {
+    thread_pool_->Schedule([this] {
```

Comment:
What is the reason for doing this asynchronously? Does it take a long time to send a log event?

I would expect a couple of milliseconds max. That shouldn't be a problem for the user since this is just called once.

Reply:
Remove async call for now, will modify the MetricsUploader itself to make async calls if needed in future